### ISSUE TITLE
[$20][MSFT-264] Keyboard access tab operational missing

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2911]
**Steps to Reproduce**
1) Go to the detail page 'https://www.topcoder.com/challenges/30095347?tab=details'
2) Find 'ABOUT US' in footer of the page (use virtual key to check)
3) Use tab to navigate from virtual keyboard
4) Observe the tab between instagram icon and support button are missing 
5) tab after instagram icon hit enter no action perform then next tab support button popup.

**Expected Result**
Every navigate tab from keyboard must be perform some action

**Actual Result**
Missing tab action while navigate via keyboard

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: 
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=8ab28edc%2Dd11f%2D4bb3%2Db9cf%2Da743b86a634f&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fkmurti%2FAccessibility%2F9

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096153 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096153 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by asadath1395** [comment]
@veshu This is happening because of an iframe created at ```src/client/index.jsx``` in ```authenticate``` function. There is also a TODO in that which says ```TODO: The iframe injected into the page by this call turns out to be visible as a tiny nob (~3x3 px). It has html ID `tc-accounts-iframe (as specified in this call). We should hide it by adding the proper side in our stylesheet, or to figure out, why it is not hidden by default.``` Is is safe to remove that iframe?

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096153 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096715 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096715 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096715 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
@asadath1395, are you still working on this or is it ready for code review?

**Comment by sadathanwar17** [comment]
Its ready for review

**Comment by veshu** [comment]
@asadath1395 Please reference the PR.

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL  : https://test-community-app.topcoder-dev.com/challenges/30055275

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by ajefts** [comment]
Verified in prod.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096715<br/><br/>```This is an automated message for crazyk via Topcoder X```
